src/redux/reservations/postReservations.js

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { getToken } from '../auth/auth';
+
+export const postReservations = createAsyncThunk(
+  'reservations',
+  async (object) => {
+    await fetch('http://127.0.0.1:3000/booking', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: getToken(),
+      },
+      body: JSON.stringify({ booking: object }),
+    });
+  },
+);
+
+export default postReservations;
```

Comment:
This request is not properly handled in the slice, for the next time make sure to add cases to handle if the request is fulfilled, pending and rejected :+1:

Reply:
Okay. Thank you for the heads up. I will implement it :+1: